tests/by-util/test_ls.rs

```diff
@@ -3580,6 +3580,58 @@ fn test_ls_dired_recursive() {
         .stdout_contains("//DIRED-OPTIONS// --quoting-style");
 }
 
+#[test]
```

Comment:
Did you notice that the positions are slightly different than those from GNU ls when running the commands of this test manually?
```
uutils ls
//DIRED// 56 58 100 102 163 165 207 216 277 278 320 322

GNU ls
//DIRED// 57 59 101 103 161 163 202 211 269 270 309 311
```

Reply:
yeah, we have some extra spaces in ls